provisioner/terraform/executor.go

```diff
@@ -242,6 +242,7 @@ func (e *executor) init(ctx, killCtx context.Context, logr logSink) error {
 		"init",
 		"-no-color",
 		"-input=false",
+		"-json",
```

Comment:
@dannykopping I added json logging to `init` here.

It was added in terraform [v1.9.0](https://github.com/hashicorp/terraform/releases/tag/v1.9.0).

> terraform init now accepts a -json option. If specified, enables the machine readable JSON output. (https://github.com/hashicorp/terraform/pull/34886)


I should add a version check though to only do this if the tf version is >1.9.0